src/modules/upcoming-events.module/module.html

```diff
@@ -1,7 +1,34 @@
 {{ require_css(get_asset_url('../../upcoming-events.css')) }}
 {{ require_js(get_asset_url('../../upcoming-events.js'), 'footer') }}
 
-<div class="upcoming-events">
-    <h2>Upcoming Events</h2>
+{%- macro outputColorIfSet(color) -%}
+  {%- if color.color -%}
+    rgba({{color.color|convert_rgb}}, {{color.opacity/100}})
+  {%- else -%}
+    inherit
+  {%- endif -%}
+{%- endmacro -%}
+
+<div class="upcoming-events"
+     style="background-color:{{outputColorIfSet(module.background_color)}};
+            color:{{outputColorIfSet(module.text_color)}};">
+     {% require_css %}
+       <style>
```

Comment:
By scoped here, I meant something that will be unique to the module's instance, which can be used via `{{ name }}` (which will output something like, `module_1234567890`). Scoping to `.upcoming-events` will cause two instances of the upcoming events module on the same page to collide with each other since they share the same `module.html` markup, i.e.:

```
both module backgrounds will be #fff :

> module A (module.background_color == #ccc)
> module B (module.background_color == #fff)
```

To solve for this, we've leaned on something like this in the past to "prefix" out selectors within inline stylesheets in <style> tags:
```css
 #hs_cos_wrapper_{{ name }} .event-card {
  background-color: ...;
}
```
which leverages an auto-generated id that gets wrapped around the module on render and effectively targets multiple instances of the same module individually, so changing the styling of `module A` doesn't effect `module B`.

I don't think this is a _huge_ deal, and maybe other folks disagree that it's something we should worry about, as I'm not sure how common it would be to display the upcoming events module more than once on the same piece of content, but given changes in #6 , there's at least app support for it and I thought I'd raise the question.